compair/tests/api/test_users.py

```diff
@@ -759,6 +760,208 @@ def test_get_current_user_course_list(self):
+            url = '/api/users/'+ student1.uuid +'/lti/users'
+            rv = self.client.get(url)
+            self.assert403(rv)
+
+        # test admin
+        with self.login('root'):
+            # test invalid data
+            url = '/api/users/999/lti/users'
+            rv = self.client.get(url)
+            self.assert404(rv)
+
+            # test list content and sort order: by user_id (asc)
+            url = '/api/users/'+ student1.uuid +'/lti/users'
+            rv = self.client.get(url)
+            self.assert200(rv)
+            self.assertEqual(2, len(rv.json['objects']))
+            self.assertEqual(student1.uuid, rv.json['objects'][0]['compair_user_id'])
+            self.assertEqual(lti_consumer.uuid, rv.json['objects'][0]['lti_consumer_id'])
+            self.assertEqual(lti_user1.user_id, rv.json['objects'][0]['lti_user_id'])
+            self.assertEqual(student1.uuid, rv.json['objects'][1]['compair_user_id'])
+            self.assertEqual(lti_consumer.uuid, rv.json['objects'][1]['lti_consumer_id'])
+            self.assertEqual(lti_user2.user_id, rv.json['objects'][1]['lti_user_id'])
+
+    def test_delete_user_lti_user(self):
+
+        # setup
+        lti_data = LTITestData()
+        lti_consumer = lti_data.create_consumer()
+
+        student1 = self.data.get_authorized_student()
+        lti_user1 = lti_data.create_user(lti_consumer, SystemRole.student, student1)
+        lti_user2 = lti_data.create_user(lti_consumer, SystemRole.student, student1)
+
+        student2 = self.data.create_normal_user()
+        lti_user1_2 = lti_data.create_user(lti_consumer, SystemRole.student, student2)
+
+        # test login required
+        url = '/api/users/'+ student1.uuid +'/lti/users/'+ lti_user2.uuid
+        rv = self.client.delete(url)
+        self.assert401(rv)
+
+        # test non-admin
+        with self.login(self.data.get_authorized_instructor().username):
+            url = '/api/users/'+ student1.uuid +'/lti/users/'+ lti_user2.uuid
+            rv = self.client.delete(url)
+            self.assert403(rv)
+
+        with self.login(self.data.get_authorized_ta().username):
+            url = '/api/users/'+ student1.uuid +'/lti/users/'+ lti_user2.uuid
+            rv = self.client.delete(url)
+            self.assert403(rv)
+
+        with self.login(self.data.get_authorized_student().username):
+            url = '/api/users/'+ student1.uuid +'/lti/users/'+ lti_user2.uuid
+            rv = self.client.delete(url)
+            self.assert403(rv)
+
+        # test admin
+        with self.login('root'):
+            # test invalid user
+            url = '/api/users/999/lti/users/'+ lti_user1.uuid
+            rv = self.client.delete(url)
+            self.assert404(rv)
+
+            # test valid
+            url = '/api/users/'+ student1.uuid +'/lti/users/'+ lti_user1.uuid
+            rv = self.client.delete(url)
+            self.assert200(rv)
+            self.assertEqual(rv.json['success'], True)
```

Comment:
check that the lti user is unlinked...

Reply:
How do I do that without calling another endpoint?

Reply:
query database